src/cdk-constructs/IxBucket.ts

```diff
@@ -12,34 +12,20 @@ export class IxBucket extends Bucket {
     id: ConstructId,
     props: ConstructProps = {},
   ) {
-    let bucketProps: ConstructProps = {}
+    const bucketProps: ConstructProps = {
+      blockPublicACLs: true,
+      cdk: {
+        bucket: {
+          enforceSSL: true,
+          ...props.cdk?.bucket,
+        },
+        ...props.cdk,
+      },
+      ...props,
```

Comment:
Same with this one, in this case `...props` should go before `cdk` (but still after `blockPublicACLs: true`) so that a user can't accidently wipe all the defaults of the cdk prop by using a props value of something like `{cdk: {someRandomProperty: true}}` and having it wipe the `enforceSSL: true` default.